xt-orphaned-directives/main.cpp

```diff
@@ -275,5 +275,5 @@ int main() {
   else
     printf ("Succeeded\n");
 
-  return 0;
+  return OUT + num_tests != 1;
```

Comment:
I think this one should be changed to:

```suggestion
  return (OUT + num_tests != 1) ? 1 : 0;
```

Reply:
Given that C states for the comparison:

> Each of the operators yields 1 if the specified relation is true and 0 if it is false. The result has type **int**.

I don't think this change gains anything. But if we want to go this route, I think it makes more sense to use something like:
```suggestion
  return (OUT + num_tests == 1) ? EXIT_SUCCESS : EXIT_FAILURE;
```
which at least has the advantage of being clearer and using pre-defined values (from stdlib.h).

Reply:
Oh sorry, maybe I was not clear (or wrong). 
Rurning `OUT + num_tests != 1;`  will return `0` when `OUT + num_tests` is different of `1`. But we want the exit code to return `1`. So we can do ` return `(OUT + num_tests == 1)`(I don't mind, just not)`OUT + num_tests != 1` as in the original code. 

The ternary was just to add more clarity, be we can remove it.